packages/init/src/tasks/addDependencies.ts

```diff
@@ -0,0 +1,43 @@
+import { execa } from 'execa';
+import ora from 'ora';
+
+import logger from '../utils/logger.js';
+
+const dependencies = [
+  'webpack',
+  'terser-webpack-plugin',
+  'babel-loader',
+  '@callstack/repack',
+];
+
+/**
+ * Installs dependencies required by Re.Pack using the specified package manager
+ *
+ * @param packageManager yarn or npm
+ */
+export default async function addDependencies(packageManager: 'yarn' | 'npm') {
+  let installCommand: string;
+
+  if (packageManager === 'yarn') {
+    installCommand = 'add';
+  } else {
+    installCommand = 'install';
+  }
+  const deps = dependencies.join(' ');
+  const command = `${packageManager} ${installCommand} -D ${deps}`;
```

Comment:
Question: does `-D` flag work for npm?

Reply:
yes, it's mentioned in the docs: https://docs.npmjs.com/cli/v8/commands/npm-install